crates/gpui/src/platform.rs

```diff
@@ -252,6 +252,16 @@ pub enum Decorations {
     },
 }
 
+/// A type to describe which GPU to prefer.
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash, Default)]
+pub enum GPUPreference {
```

Comment:
Can we follow Rust casing conventions here?

```suggestion
pub enum GpuPreference {
```